CODEOWNERS

```diff
@@ -1 +1 @@
-* @dmcgowan @jzelinskie @jdolitsky @mikebrow @stevvooe @vbatts
+* @dmcgowan @jzelinskie @jonjohnsonjr @jdolitsky @mikebrow @stevvooe @vbatts
```

Comment:
unrelated question: could this line in the file just be `* @opencontainers/distribution-spec-maintainers` , so we don't have to manage this list in several places?